doc/modules/rbl.md

```diff
@@ -556,3 +556,10 @@ symbols_prefixes {
   subject_digest = "SUBJECT_DIGEST"
 }
 ~~~
+
+If there need to combine multiply selectors in one query you can set `selector_flatten` to `false` to achieve it. For example to create query to RBL like: `from-sha256.rcpt-sha256`:
```

Comment:
s/multiply/multiple/

Reply:
If there **is** **a** need to or `If one needs to`

Reply:
Fixed